src/lib/Scenes/Favorites/Components/Shows/index.tsx

```diff
@@ -75,23 +75,7 @@ export default createPaginationContainer(
           shows(first: $count, after: $cursor) @connection(key: "SavedShows_shows") {
             edges {
               node {
-                id
-                is_followed
-                _id
-                name
-                partner {
-                  ... on Partner {
-                    name
-                  }
-                  ... on ExternalPartner {
-                    name
-                  }
-                }
-                href
-                status
-                images(size: 1) {
-                  url
-                }
+                ...SavedShowItemRow_show
```

Comment:
This is the core change – defining the data for the `SavedShowItemRow` "closer" to the code that uses it. The parent component becomes more focused while the child component gets the benefits of generated types.